store/src/main/java/com/dropbox/android/external/store4/StoreBuilder.kt

```diff
@@ -33,12 +33,29 @@ import kotlinx.coroutines.flow.flow
 @ExperimentalCoroutinesApi
 interface StoreBuilder<Key, Output> {
     fun build(): Store<Key, Output>
+    /**
+     * A store multicasts same [Output] value to many consumers (Similar to RxJava.share()), by default
+     *  [Store] will open a global scope for management of shared responses, if instead you'd like to control
+     *  the scope that sharing/multicasting happens in you can pass a @param [scope]
+     *
+     *   @param scope - scope to use for sharing
+     */
     fun scope(scope: CoroutineScope): StoreBuilder<Key, Output>
+
+    /**
+     * controls eviction policy for a store cache, use [MemoryPolicy.MemoryPolicyBuilder] to configure a TTL
+     *  or size based eviction
+     *  Example: MemoryPolicy.builder().setExpireAfterWrite(10).setExpireAfterTimeUnit(TimeUnit.SECONDS).build()
+     */
     fun cachePolicy(memoryPolicy: MemoryPolicy?): StoreBuilder<Key, Output>
+
+    /**
+     * by default a Store caches in memory with a default policy of max items = 16
+     */
     fun disableCache(): StoreBuilder<Key, Output>
 
     /**
-     * Connects a (non-[Flow]) source of truth that is accessible via [reader], [writer] and
+     * Connects a (non-[flow]) source of truth that is accessible via [reader], [writer] and
```

Comment:
I think capitalized is correct here. we want to reference the Flow Class not a factory function

Reply:
fixed